src/main/java/swift/logic/commands/CommandResult.java

```diff
@@ -18,33 +18,37 @@ public class CommandResult {
     private final boolean exit;
 
     /** The application should switch to contacts tab. */
-    private final boolean isListContact;
+    private final boolean isContactCommand;
 
     /** The application should switch to tasks tab. */
-    private final boolean isListTask;
-
-    /** The application should display all tasks with associated contacts. */
-    private final boolean isAddTask;
+    private final boolean isTaskCommand;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean isListContact,
-                         boolean isListTask, boolean isAddTask) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean isContactCommand,
+                         boolean isTaskCommand) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
         this.exit = exit;
-        this.isListContact = isListContact;
-        this.isListTask = isListTask;
-        this.isAddTask = isAddTask;
+        this.isContactCommand = isContactCommand;
+        this.isTaskCommand = isTaskCommand;
     }
 
     /**
      * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
      * and other fields set to their default value.
      */
     public CommandResult(String feedbackToUser) {
-        this(feedbackToUser, false, false, false, false, false);
+        this(feedbackToUser, false, false, false, false);
```

Comment:
this will get us flagged for code quality. It's unreadable. What do these magic booleans mean? Create constants for them at the top of the java file, please